x/proof/types/expected_keepers.go

```diff
@@ -7,21 +7,29 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
+	apptypes "github.com/pokt-network/poktroll/x/application/types"
 	sessiontypes "github.com/pokt-network/poktroll/x/session/types"
 )
 
 type SessionKeeper interface {
 	GetSession(context.Context, *sessiontypes.QueryGetSessionRequest) (*sessiontypes.QueryGetSessionResponse, error)
+	GetBlockHash(ctx context.Context, height int64) []byte
 }
 
 // AccountKeeper defines the expected interface for the Account module.
 type AccountKeeper interface {
-	GetAccount(context.Context, sdk.AccAddress) sdk.AccountI // only used for simulation
-	// Methods imported from account should be defined here
+	GetAccount(context.Context, sdk.AccAddress) sdk.AccountI
 }
 
 // BankKeeper defines the expected interface for the Bank module.
 type BankKeeper interface {
 	SpendableCoins(context.Context, sdk.AccAddress) sdk.Coins
-	// Methods imported from bank should be defined here
+	DelegateCoinsFromAccountToModule(ctx context.Context, senderAddr sdk.AccAddress, recipientModule string, amt sdk.Coins) error
+	UndelegateCoinsFromModuleToAccount(ctx context.Context, senderModule string, recipientAddr sdk.AccAddress, amt sdk.Coins) error
+}
```

Comment:
Didn't you remove the `BankKeeper` as a dependency in `x/proof/module/module.go` in the `ModuleInputs`. Or am I missing something?

Reply:
Yes it's not needed by the Proof keeper itself but seems useful for integration tests:

https://github.com/pokt-network/poktroll/pull/406#discussion_r1530163406